### PR TITLE
feat: use payment request settings account keys

### LINKED ISSUE #1846: [settings] Stripe "payment request" button preview
## Description

The button preview on the "Payment request" customization page is not working.
We need to pass the correct parameters to the `loadStripe` function, in order for it to work properly.

This is the page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&area=payment_requests

## Acceptance criteria
- The "Payment request" customization page shows the Stripe button preview
- When the settings change, the preview also updates

### PR DESCRIPTION
**Pull request by frosso**
# Changes proposed in this Pull Request:

Almost forgot about this little thing :D 
We're using some mocked data for the "Payment request" button.
Let's use the store's account keys.
Screen Shot 2021-10-08 at 4 22 43 PM

(when loaded with HTTP, instead of HTTPS:)

Screen Shot 2021-10-08 at 4 33 05 PM

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure your account is connected
- Start your Jurasic Tube tunnel
- Ensure you're using HTTPS to access your site
- Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&area=payment_requests`
- Button should now appear!

### CODE REVIEW

File: client/entrypoints/payment-request-settings/payment-request-section.js [hunk 1]
```diff
@@ -115,22 +119,19 @@ const PaymentRequestsSection = () => {
...
-		return loadStripe( stripeSettings.publishableKey, {
-			stripeAccount: stripeSettings.accountId,
-			locale: stripeSettings.locale,
-		} );
-	}, [] );
+		return loadStripe(
+			publishableKey || testPublishableKey || 'pk_test_123',
+			{
+				stripeAccount: accountId || '0001',
+				locale: 'en',
```
Comment (waclawjacek) [inline client/entrypoints/payment-request-settings/payment-request-section.js:131]: Should we ever use default values? Shouldn't it fail if a value is missing?

**Review by waclawjacek — COMMENTED** [review]
Tested and works great! No button on HTTP, button's there on HTTPS.

One question posted in comments.

WDYT: Should we also (this is a matter for another issue) add some more info on the circumstances under which the button is not displayed (such as the requirement to use HTTPS)? Maybe add a section to the docs and link to it in the notice?

File: client/entrypoints/payment-request-settings/payment-request-section.js (hunk 1 above)
Comment (frosso) [reply client/entrypoints/payment-request-settings/payment-request-section.js:131]: The default values also make it work. So I was wondering to either just keep the default values (closing the PR with no changes) or using the merchant's keys 🤷

**Comment by frosso** [comment]
> WDYT: Should we also (this is a matter for another issue) add some more info on the circumstances under which the button is not displayed (such as the requirement to use HTTPS)? Maybe add a section to the docs and link to it in the notice?

When a website is not loaded via HTTPS, we do show a notice that says:

> Stripe is enabled, but a SSL certificate is not detected

We show a similar message in WCPa - however, I also think that an additional messaging might be useful to some merchants.
I don't think a site without SSL would be considered "secure" for Stripe, so we could definitely add something to the documentation once we'll release it :)

File: client/entrypoints/payment-request-settings/payment-request-section.js (hunk 1 above)
Comment (waclawjacek) [reply client/entrypoints/payment-request-settings/payment-request-section.js:131]: I see! Thanks! If there is no harm in using real credentials, it seems like a good idea to use them.

**Review by waclawjacek — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**